### ISSUE TITLE
Streamline the Homepage

### ISSUE THREAD

**Issue by RonitNath** [ocf/ocfstatic#270]
Add big buttons linking to "Create Account" (exists), "Join Staff", "Staff Hours, Get Help", "Host Your Website", "Contact Us", "How to Print"

These should be prominently displayed on the website's landing page. The purpose is to increase accessibility.

The links should be removed, or moved to another location, and all three sections below it should be moved up.

The above links should send people to pages which are designed with extreme beginners in mind.
